packages/RatPack.package/RatPackSetup.class/class/page2.st

```diff
@@ -0,0 +1,23 @@
+pages
+page2
+	^HelpTopic 
+		title: '2. Squeak and Postgresql'
+		contents: 'Setup and run Postgresql server (tested: 9.4 on windows)
+
+	1. get package Cryptography (MD5 class is needed) http://www.squeaksource.com/Cryptography
```

Comment:
where is this from? I don't remember manually installing these two packages. Don't they come with RatPack?

Reply:
I just copied everything we got from the wiki. no time for verification.